Beta_Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -0,0 +1,1648 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+
+import json
+import requests
+from distutils.util import strtobool
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+
+USERNAME = demisto.params().get('credentials').get('identifier')
+PASSWORD = demisto.params().get('credentials').get('password')
+TOKEN = demisto.params().get('token')
+SERVER = demisto.params()['url'][:-1] if (demisto.params()['url'] and demisto.params()['url'].endswith('/')) \
+    else demisto.params()['url']
+USE_SSL = not demisto.params().get('unsecure', False)
+FETCH_TIME = demisto.params().get('fetch_time', '3 days')
+FETCH_THREAT_RANK = int(demisto.params().get('fetch_threat_rank', 5))
+BASE_URL = SERVER + '/web/api/v2.0/'
+HEADERS = {
+    'Authorization': 'ApiToken ' + TOKEN,
+    'Content-Type': 'application/json',
+    'Accept': 'application/json'
+}
+
+# remove proxy if not set to true in params
+if not demisto.params().get('proxy'):
+    del os.environ['HTTP_PROXY']
+    del os.environ['HTTPS_PROXY']
+    del os.environ['http_proxy']
+    del os.environ['https_proxy']
+
+''' HELPER FUNCTIONS '''
+
+
+def http_request(method, url_suffix, params={}, data=None):
+    res = requests.request(
+        method,
+        BASE_URL + url_suffix,
+        verify=USE_SSL,
+        params=params,
+        data=data,
+        headers=HEADERS
+    )
+
```

Comment:
Might be nice to log here the API request being sent, like we log the commad before the flow of the code starts (see before switch case), just don't forget to not log HEADERS var as it contains sensitive info.